container/cli.py

```diff
@@ -171,6 +172,10 @@ def commandline():
     parser.add_argument('--project', '-p', action='store', dest='base_path',
                         help=u'Specify a path to your project. Defaults to '
                              u'current working directory.', default=os.getcwd())
+    parser.add_argument('--var-files', action='store',
```

Comment:
How would multiple files work with order of inheritance? I'm going to suggest we start with just one.